core_impl/src/main/java/com/google/instrumentation/stats/MutableView.java

```diff
@@ -29,6 +30,7 @@
  */
 // TODO(songya): remove or modify the methods of this class, since it's not part of the API.
 abstract class MutableView {
+  @VisibleForTesting static final TagValue UNKNOWN_TAG_VALUE = TagValue.create("unknown/not set");
```

Comment:
Marked. The current value is from previous discussion (not sure if we've come to a conclusion on this).